python/whylogs/experimental/core/metrics/embedding_metric.py

```diff
@@ -166,10 +167,26 @@ def _update_submetrics(self, submetric: str, data: PreprocessedColumn) -> None:
             self.submetrics[submetric][key].columnar_update(data)
 
     def columnar_update(self, data: PreprocessedColumn) -> OperationResult:
-        if data.numpy.ints is None and data.numpy.floats is None:
+        if data.numpy.ints is None and data.numpy.floats is None and data.pandas.vectors is None:
             return OperationResult.ok(0)
 
-        matrices = [X for X in [data.numpy.ints, data.numpy.floats] if X is not None]
+        converted_numpy_ints = None
+        if data.numpy.ints is not None and len(data.numpy.ints) > 0:
+            converted_numpy_ints = data.numpy.ints
+            if is_not_stub(pd.Series) and isinstance(data.numpy.ints, pd.Series) and not data.numpy.ints.empty:
```

Comment:
Hmmm...

Reply:
Yes, there is a more general fix for this that I was afraid of making (due to PreprocessedColumn magic) outside of this case. I'll add a small PR just for that change for you and Jamie to look at.